zoo/evaluation/scenarios/cutin_escape_1/traffic/flowcar1.rou.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/routes_file.xsd">
+
+  <vType id="flow-car" accel="3.0" decel="1.0" minGap="0" maxSpeed="5" sigma="0.5" lcStrategic="0.0" lcCooperative="0.0" lcSpeedGain="0.0" lcKeepRight="0.0" 
```

Comment:
Where are these coming from? `minGap="0" and `lcCooperative=0.0` is inviting collisions.